Source/WSTagsField.swift

```diff
@@ -160,6 +169,15 @@ open class WSTagsField: UIView {
         }
     }
     
+    public var attributedText: NSAttributedString? {
```

Comment:
useful if you want to do custom highlighting in validations like we do when the string is too long